client/my-sites/comments/comment/index.jsx

```diff
@@ -42,23 +42,35 @@ export class Comment extends Component {
 	};
 
 	componentWillReceiveProps( nextProps ) {
-		if ( nextProps.isBulkMode && ! this.props.isBulkMode ) {
-			this.setState( { isExpanded: false } );
+		const { isBulkMode: wasBulkMode, isPostView: wasPostView } = this.props;
+		const { isBulkMode, isLoading, isPostView } = nextProps;
+		const { isExpanded: wasExpanded, isReplyVisible: wasReplyVisible } = this.state;
+
+		let isExpanded = wasExpanded;
+		if ( isPostView && ( ! wasPostView || wasBulkMode !== isBulkMode ) ) {
```

Comment:
This whole block for inferring `isExpanded` value looks confusing to me. Why do we need `wasPostView` to determine it, isn't it enough to know that we are currently in post view in order to show the collapsed version? (same goes for `wasBulkMode`). 

Also if we factor out `isLoading`, since I don't think it's needed anymore because we want to show expanded placeholders, it looks to me like all of this could be simplified to:

`const isExpanded = ! isPostView && ! isBulkMode;`